x-pack/plugins/security_solution/public/common/components/event_details/insights/related_alerts_by_session.tsx

```diff
@@ -79,13 +82,31 @@ export const RelatedAlertsBySession = React.memo<Props>(
       } else if (isEmpty && state !== 'loading') {
         return SESSION_EMPTY;
       }
+      const ensureOnlyRuleProviders: DataProvider = {
+        enabled: true,
+        id: ALERT_RULE_UUID,
```

Comment:
I'm weary of using the _s word_ in user-facing UI, but AFIK `event.kind: "signal"` is technically (still) the ECS field / value that distinguishes an alert from an event.

It's true that the `kibana.alert.rule.uuid` field will, for non-legacy alerts, exist in every alert, but that feels like an implementation detail of the current detection engine. (Legacy "signal" alerts generated from older versions of the detection engine don't have the `kibana.alert.rule.uuid` field.)

- @paulewing would you be willing to confirm that `event.kind: "signal"` is still the ECS-compliant field that differentiates alerts from events?

- @paulewing even if the answer to the previous question is "yes",  are we ok with the term `signal` appearing in the UI as filter criteria, or would it be better to test for the existence of `kibana.alert.rule.uuid`, as this would avoid using the value `signal` in the filter?